src/main/java/frc/robot/Constants.java

```diff
@@ -93,4 +93,15 @@ public static enum Mode {
     /** Replaying from a log file. */
     REPLAY
   }
+
+  static {
+    RobotConfig config;
+    try {
+      config = RobotConfig.fromGUISettings();
+      // Check if the GUI settings match the constants
+      PPUtil.compareConfigs(config, PP_CONFIG);
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
what happens overall if we get an excerption here? just printing the stack trace probably isn't everything? is there some other action tha tneeds to happen if you get an exception check the config values vs the gui?

Reply:
Sounds good. I'll warn that it couldn't load the configuration from the GUI. Since we aren't using it, it's not a huge deal, but it should give some feedback.